frontend/src/solo-types/apiDoc.ts

```diff
@@ -174,6 +174,13 @@ export const defaultApiDoc: ApiDocument = {
   sdn: "M3030012341234"
 };
 
+export const defaultApiResponse: PaginatedApiResponse<ApiDocument[]> = {
+  results: [defaultApiDoc],
+  count: 100,
+  next: 1,
+  previous: 2
+};
+
```

Comment:
Have you used an api before that returns like this? I would expect it to be `currentPage`, `pageSize` and `results`. Don't care either way just surprised.

Reply:
This is how Django Rest Framework default pagination structures results.